packages/commonjs-shim/README.md

```diff
@@ -0,0 +1,49 @@
+[npm]: https://img.shields.io/npm/v/@rollup/plugin-commonjs-shim
+[npm-url]: https://www.npmjs.com/package/@rollup/plugin-commonjs-shim
+[size]: https://packagephobia.now.sh/badge?p=@rollup/plugin-commonjs-shim
+[size-url]: https://packagephobia.now.sh/result?p=@rollup/plugin-commonjs-shim
+
+[![npm][npm]][npm-url]
+[![size][size]][size-url]
+[![libera manifesto](https://img.shields.io/badge/libera-manifesto-lightgrey.svg)](https://liberamanifesto.com)
+
+# @rollup/plugin-commonjs-shim
+
+🍣 A Rollup plugin to replace CJS syntax like `__filename`, `__dirname`, ... for esm output bundles.
+
+## Requirements
+
+This plugin requires an [LTS](https://github.com/nodejs/Release) Node version (v14.0.0+) and Rollup v2.0+.
+
+## Install
+
+Using npm:
+
+```console
+npm install @rollup/plugin-commonjs-shim --save-dev
+```
+
+## Usage
+
+Create a `rollup.config.js` [configuration file](https://www.rollupjs.org/guide/en/#configuration-files) and import the plugin:
+
+```typescript
+import commonjsShim from '@rollup/plugin-commonjs-shim';
```

Comment:
We're technically not shimming CJS; we're shimming ESM so it can use CJS things. A more accurate name for this would `esm-shim`. I know that's kind of a PITA to change at this point but it's important to convey meaning

Reply:
No problem, since I have many principles by which I live and also program, it happens more often that I have to refactor code due to less than optimal decisions.

Reply:
Moreover, you are absolutely right with your statement, since it is a function extension for esm and not cjs.